src/vscode/commands/Repl.ts

```diff
@@ -63,6 +67,35 @@ export async function inlineEval(state: ExtensionState) {
     })
 }
 
+export async function startReplAndAttach(state: ExtensionState, ctx: vscode.ExtensionContext) {
+    const cmd = vscode.workspace.getConfiguration('alive').swank.startCommand;
+    const outputAndAttach = (data: string) => {
+        outputChannel.appendLine(data)
+
+        if (data.includes("Swank started at port")) {
+            attachRepl(state, ctx)
+        }
+    }
+    const spawn = () => {
+        child = childProcess.spawn(cmd[0], cmd.slice(1))
```

Comment:
Add child.on('error', ...) that displays the error message.
Add child.on('exit', ...) that resets child to null.